f/medical/fn_famAddHealAction.sqf

```diff
@@ -16,8 +16,6 @@ private _healMedicTime = 4.5; // Action Duration
 
 // Starting Code
 private _healCodeStart = { 
-	params ["_target", "_caller", "_actionId", "_arguments"]; 
```

Comment:
Don't we still need `_caller` from most of these `params` calss that have been removed? (this one + a couple below)

Reply:
We do, but in `BIS_fnc_holdActionAdd`, all these things are already defined, as if `params` has already been run with those names. We don't need to define them again.

Reply:
Oh right, I forget how weird SQF scoping rules are